metrics.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+)
+
+var (
+	reportGenerationTotal = promauto.NewCounter(prometheus.CounterOpts{
+		Namespace: common.MetricsNamespace,
+		Subsystem: common.MetricsSubsystem,
+		Name:      "report_generation_total",
+		Help:      "Total number of report generation attempts",
+	})
```

Comment:
I'm not sure I understand what this metric is trying to measure, but looking at the code it seems covered by `unitMetricsTotal`.